src/Bicep.RegistryModuleTool.UnitTests/ModuleValidators/ModulePathValidatorTests.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Bicep.RegistryModuleTool.Exceptions;
+using Bicep.RegistryModuleTool.ModuleValidators;
+using Bicep.RegistryModuleTool.TestFixtures.MockFactories;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO.Abstractions.TestingHelpers;
+using static FluentAssertions.FluentActions;
+
+namespace Bicep.RegistryModuleTool.UnitTests.ModuleValidators
+{
+    [TestClass]
+    public class ModulePathValidatorTests
+    {
+        [TestMethod]
+        public void ValidateModulePath_InvalidPath_Throws()
+        {
+            const string currentDirectory = "/foo/bar";
+            var fileSystem = new MockFileSystem();
+
+            fileSystem.AddDirectory(currentDirectory);
+            fileSystem.Directory.SetCurrentDirectory(fileSystem.Path.GetFullPath(currentDirectory));
+
+            Invoking(() => ModulePathValidator.ValidateModulePath(fileSystem))
+                .Should()
+                .Throw<InvalidModuleException>()
+                .WithMessage(@"Could not find the ""modules"" folder in the path ""C:\foo\bar"".");
```

Comment:
Does the message use a Windows-style path on other OSes?

Reply:
Good catch. The test would fail on other OSes. Changed it to use `fileSystem.Directory.GetCurrentDirectory()`.